services/ui-src/src/verbiage/forms/adminBannerForm.ts

```diff
@@ -4,6 +4,40 @@ export const form = {
     mode: "onChange",
   },
   fields: [
+    {
+      id: "abf-radio",
+      type: "choiceList",
+      props: {
+        name: "Choice List Radio Name",
+        type: "radio",
+        label: "Choice List Radio Label",
+        choices: [
+          { label: "Choice 1", value: "A", defaultChecked: true },
+          { label: "Choice 2", value: "B" },
+          { label: "Disabled choice 3", value: "C", disabled: true },
```

Comment:
I've talked to design and we will not be using `defaultChecked` or `disabled` on ChoiceList choices, so no need to support those for MVP. 👍